model/project.go

```diff
@@ -759,6 +759,17 @@ func (p *Project) FindTaskForVariant(task, variant string) *BuildVariantTaskUnit
 	return nil
 }
 
+func (bv *BuildVariant) InDisplayTask(taskName string) string {
```

Comment:
This seems like it should return a bool.

Reply:
It did at one point. I renamed it